charts/stan/values.yaml

```diff
@@ -6,7 +6,7 @@
 images:
   init:
     repository: quay.io/astronomer/ap-base
-    tag: 3.15.0-3
+    tag: 3.15.2
```

Comment:
Like my pervious comment: you told me to use new version 3.15.3 yesterday, shouldn't we include here as this is the latest one you've built ?